src/main/resources/interface-specification.yml

```
@@ -2386,6 +2548,80 @@ components:
         - eventType
         - objectType
         - objectId
+    Key:
```

what about
```suggestion
    JWK:
```
?
So we can use `Key` for the actual key model